tests/doc-links.spec.ts

```diff
@@ -0,0 +1,218 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  editInstance,
+  randomInstanceName,
+} from "./helpers/instances";
+import {
+  createProject,
+  deleteProject,
+  randomProjectName,
+} from "./helpers/projects";
+import { validateDocumentationLink } from "./helpers/doc-links";
+import { visitServerSettings } from "./helpers/server";
+
+test("Ensure the documentation link text and link targets are present: Server Settings > Override", async ({
+  page,
+}) => {
+  await visitServerSettings(page);
```

Comment:
I think for 5.0 we should skip this test, as the generated documentation only started shipping with the newer versions of LXD. You can find examples in other tests how to do this.

Reply:
I think we need to skip the instance config and project config tests for 5.0 as well since the required api extension `metadata_configuration` is not available for LXD v5.0-stable, see [doc](https://documentation.ubuntu.com/lxd/en/stable-5.0/api-extensions/).

fyi @Kxiru API extension is how we determine if certain features are available for a specific LXD installation. The available API extensions are different per LXD version. You can check the `useSupporteddFeatures` hook and see where it gets used in the code base to get a sense on how it works.

Reply:
@mas-who , how can you tell what api extension is used and when/where?

I'll make the other tests also only available for LXD 5++